MeNext/Playing/Poller.cs

```diff
@@ -28,6 +28,7 @@ public class Poller
 			// TODO: Fix this warning?
 			await Task.Run(async () =>
 			{
+				var api = new BackendAPI("http://192.168.8.242:8080");
```

Comment:
We should prolly move this to a config file out of version control, so each of us can use our own IP. No need for this release though.